### PR TITLE
Feature/locker 1216

### PR DESCRIPTION
**Pull request by muralikuppoor**

### CODE REVIEW

File: airside-security-console-ui/src/app/views/customers/customers.component.ts [hunk 1]
```diff
@@ -30,6 +33,68 @@ export class CustomersComponent implements OnInit, CredentialsSearchListener, Fo
   page5 = 5;
   warningMessage = 'Do you really want to delete the credential?';
 
+  oAuthClientDetails: OAuthClientDetailsTO [] = [{
+    clientId: "user_client1",
+    clientSecret: "$2a$09$kR47DnltbB5VFL0YSnm1Luvr15/4236347654745786585865",
```
Comment (rajairside) [inline airside-security-console-ui/src/app/views/customers/customers.component.ts]: Please remove these hard coded hashes

File: airside-security-console-ui/src/app/forms/customer-form/customer-form.component.ts [hunk 2]
```diff
@@ -47,6 +48,54 @@ export class CustomerFormComponent implements OnInit {
...
+    'code.read',
+    'info.travel.read',
+    'apis',
+    'mrch.catalog',
+    'mrch.authorize'
+  ];
+  rolesListOAuth: string [] = ['ROLE_USER', 'ROLE_DOCS.READ', 'ROLE_DOCS.WRITE',
+    'ROLE_SHARE.READ', 'ROLE_SHARE.WRITE', 'ROLE_INFO.TRAVEL.READ', 'ROLE_INFO.TRAVEL.WRITE',
+    'ROLE_CMRC.CODE.USER', 'ROLE_CMRC.CODE.READ', 'ROLE_CMRC.CODE.MANAGE', 'ROLE_KEY',
+    'ROLE_APIS', 'ROLE_DOCS.DELETE', 'ROLE_SHARE.DELETE', 'ROLE_ADMIN.READ', 'ROLE_ADMIN.WRITE', 'ROLE_ADMIN.DELETE'];
```
[the remark this answers is not in the document]
Comment (muralikuppoor) [reply airside-security-console-ui/src/app/forms/customer-form/customer-form.component.ts]: Agreed! Will populate from backend next sprint.

File: airside-security-console-ui/src/app/forms/customer-form/customer-form.component.ts [hunk 3]
```diff
@@ -47,6 +48,54 @@ export class CustomerFormComponent implements OnInit {
...
+  clientId: string;
+  inValidClientId: boolean;
+  inValidClientIdLength: boolean;
+  secret: string;
+  inValidSecret: boolean;
+  inValidSecretLen: boolean;
+  invalidSecretFormat: boolean;
+  changeSecret: boolean;
+  authGrantList: string [] = ['authorization_code', 'implicit', 'client_credentials', 'password'];
+  scopeList: string [] = [
```
[the remark this answers is not in the document]
Comment (muralikuppoor) [reply airside-security-console-ui/src/app/forms/customer-form/customer-form.component.ts]: 👍

File: airside-security-console-ui/src/app/views/customers/customers.component.ts (hunk 1 above)
Comment (muralikuppoor) [reply airside-security-console-ui/src/app/views/customers/customers.component.ts]: Good catch! removed. I used to build the list before fetching the data from backend.

**Pull request merged**